src/OpenSage.Game/Gui/Apt/DisplayList.cs

```diff
@@ -4,11 +4,11 @@ namespace OpenSage.Gui.Apt
 {
     public sealed class DisplayList
     {
-        public Dictionary<int, DisplayItem> Items { get; }
+        public SortedDictionary<int, DisplayItem> Items { get; }
```

Comment:
DisplayItems are sorted by a depth value. Before they were just randomly in here (depending on the insertion order)